ci/run_truffle_tests.sh

```diff
@@ -47,7 +47,8 @@ run_tests $PROTOCOL_DIR/core
 
 # Hacky way of running truffle tests outside of core.
 cd $PROTOCOL_DIR/core
-$(npm bin)/truffle test ../financial-templates-lib/test/*.js --network ci
+
+$(npm bin)/truffle test $(find ../financial-templates-lib/test -name '*.js') --network ci
```

Comment:
Would this script work for the other scaffolding directories `liquidator`, `disputer` and `monitors`?

Reply:
Yeah, the reason I only added it here is that this is the only one that has a nested directory. Do you think we should go ahead and do it for all of them?

Reply:
Done.